config/foreman.migrations/180821005330_foreman_jobs_service.rb

```diff
@@ -0,0 +1,3 @@
+if answers['foreman'].is_a?(Hash) && answers['foreman']['foreman_jobs_service'].nil?
+  answers['foreman']['foreman_jobs_service'] = 'dynflowd'
```

Comment:
From testing this should just be `jobs_service`. Please also open one for Katello :)